builtin/providers/google/resource_google_project.go

```diff
@@ -31,18 +32,29 @@ func resourceGoogleProject() *schema.Resource {
 		Delete: resourceGoogleProjectDelete,
 
 		Schema: map[string]*schema.Schema{
-			"id": &schema.Schema{
+			"project_id": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-			"policy_data": &schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
The initial version of this resource didn't support create. It assumed the resource (project) already existed, and it read/computed the values from the existing resource.

I'm wondering if it's worth breaking this out into two resources (`google_project` and `google_imported_project`), or adding an `imported` attribute? The latter is simpler, but would have issues with computed vs non-computed values. Thoughts?